AirCasting/MicrophoneSession/MicrophoneManager.swift

```diff
@@ -121,7 +140,15 @@ private extension MicrophoneManager {
                                        thresholdMedium: 70,
                                        thresholdLow: 60,
                                        thresholdVeryLow: 20)
-        return try measurementStreamStorage.createSessionAndMeasurementStream(session, stream)
+        
+        measurementStreamStorage.accessStorage { [self] storage in
```

Comment:
strong capturing `[self]` is redundant (seems like `self` is not used inside too)

Reply:
Removed